### ISSUE TITLE
Database name validity issue

### ISSUE THREAD

**Issue by manjujidagi** [orangehrm/orangehrm#1800]
### OrangeHRM version

5.6

### Environment details

![Screenshot 2024-02-01 at 4 31 55 PM](https://github.com/orangehrm/orangehrm/assets/17455675/dce77b96-8a77-472f-b6a3-6a6ccf52b908)

### Describe the bug

The database name `orangehrm-3231319527` is valid name but the application says its invalid.

The reason given by application is - 'Database name should not contain special characters'.

But the database can have `-` in the name

**Comment by ManulMax** [comment]
Hi @manjujidagi ,
You are true about  _database can have - in the name_ . You can add special character to the database name. But in most database management systems it is recommended to avoid such special character in database name.

**Comment by manjujidagi** [comment]
Yes, but until that is allowed by DB providers, the application should allow it ( may be with a warning )

**Comment by tomslux** [comment]
Exactly the same Problem here... Big Hoster like HostEurope are forcing those "-" in database names... so I can't install this project on my webhost...
